integration-tests-ansible/test-project-play-rpm/packaging.sbt

```diff
@@ -28,3 +28,5 @@ rpmRelease := "1"
 rpmVendor := "DemoVendor"
 
 rpmLicense := Some("Apache-2.0")
+
+rpmEpoch := 1
```

Comment:
Out of curiosity? This do you acutally run this test? I haven't used this for a very long time and I'm not sure if we should remove it.

Reply:
Whoops, I forgot that I committed this. I mucked around with the Vagrant file (e.g., changing chef to bento) and got something half running, but tbh you could probably remove it.